applications/StructuralMechanicsApplication/tests/rigid_test/sphere_failing.mdpa

```diff
@@ -0,0 +1,879 @@
+Begin ModelPartData
```

Comment:
why did u change this?

Reply:
@RiccardoRossi this PR includes another PR, #3249 must be approved and merged first